psa-crypto/src/types/key.rs

```diff
@@ -541,10 +542,40 @@ pub enum Type {
     },
 }
 
+impl fmt::Display for Type {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Type::RawData => write!(f, "Raw data"),
+            Type::Hmac => write!(f, "HMAC key"),
+            Type::Derive => write!(f, "Derivation key"),
+            Type::Aes => write!(f, "Key for an algorithm based on AES"),
+            Type::Des => write!(f, "Key for an algorithm based on DES or 3DES"),
+            Type::Camellia => write!(f, "Key for an algorithm based on Camellia"),
+            Type::Arc4 => write!(f, "Key for the RC4 stream cipher"),
+            Type::Chacha20 => write!(
+                f,
+                "Key for the ChaCha20 stream cipher or the Chacha20-Poly1305 AEAD algorithm"
```

Comment:
Is this how it's defined in the spec? Seems sensible to just write it like the ones above `Key for an algorithm based on ChaCha20`

Reply:
Yees that's directly taken from the spec :/ https://parallaxsecond.github.io/parsec-book/parsec_client/operations/psa_key_attributes.html#chacha20-type

Reply:
Hmm, though you did "sanatize" the ones above, they're not just copy-pasted